book/P1C6_survtsk.qmd

```diff
@@ -10,119 +10,270 @@ abstract: "This chapter introduces the different survival problems and formalise
-For example, given three subjects, $\{i,j,k\}$, a relative risk prediction may predict the risk of event as $\{0.1, 0.5, 10\}$ respectively.
-From these predictions, the following types of conclusions can be drawn:
+Predicting risks is perhaps the most common survival problem and is defined as predicting a continuous rank for an individual's relative risk of experiencing the event.
+Therefore this might also be known as a 'ranking' problem or predicting 'continuous rankings'.
+In machine learning terms, this task is the problem of estimating, $g: \calX \rightarrow \Reals$.
+
+Interpretation of these rankings is more complex than might be imagined as the meaning differs between models, parametrizations, and even software implementation.
+To be consistent, in this book a larger risk value *always* corresponds to a higher risk of an event and lower values corresponding to lower risk.
+
+A potential confusion that should be avoided is conflating this risk prediction with a prediction of absolute risk.
+Risk predictions are specifically *relative risks*, which means the risks are only comparable to other observations within the same sample.
+For example, given three subjects, $\{i,j,k\}$, a risk prediction may be $\{0.5, 10, 0.1\}$ respectively.
+From these predictions, two primary types of conclusion can be drawn.
+
+1) Conclusions comparing subjects:
 
-* Conclusions comparing subjects. For example, $i$ is at the least risk; the risk of $j$ is only slightly higher than that of $i$ but the risk of $k$ is considerably higher than $j$; the corresponding ranks for $i,j,k,$ are $1,2,3$;
-* Conclusions comparing risk groups. For example, thresholding the risks at $1.0$ means that $i$ and $j$ are in a low-risk group whilst $k$ is in a high-risk group.
+* The corresponding ranks for $i,j,k$ are $2,3,1$;
+* $k$ is at the least risk and $j$ is at the highest risk;
+* The risk of $i$ is slightly higher than that of $k$ but $j$'s risk is considerably higher than both the others.
+
+2) Conclusions comparing risk groups:
+
+* Thresholding risks at $0.4$ means $k$ is at a low-risk but $i$ and $j$ are high-risk.
+* Thresholding risks at $1.0$ means $i$ and $k$ are low-risk but $j$ is high-risk.
 
 Whilst many important conclusions can be drawn from these predictions, the values themselves have no meaning when not compared to other individuals.
-Interpretation of these rankings depends on the model class (for example, PH and AFT models have opposite interpretations, @sec-models-classical) and its parametrization or implementation in specific software.
-For some higher ranking implies higher risk whereas others may assume that higher ranking implies lower risk.
-In this book, a higher ranking will always imply a higher risk of event (as in the example above).
+Similarly, the values have no meanings across research, even if observation $k$ is at low-risk according to this sample, it may be high risk compared to another.
+
```

Comment:
maybe worth adding somewhere that even large differences in relative risk might not be relevant in practice in terms of absolut risk. e.g. i has 5 times higher risk than k, but if baseline probability is .0001, the absolute risk is still low.